### PR TITLE
Switch from Mailable to Notification

### PR DESCRIPTION
**Pull request by ZeProf2Code**

### CODE REVIEW

File: src/Notifications/PersonalDataExported.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+     * The callback that should be used to build the mail message.
+     *
+     * @var \Closure|null
+     */
+    public static $toMailCallback;
+
+    /**
+     * @var string 
+     */
+    public $zipFilename;
```
Comment (freekmurze) [inline src/Notifications/PersonalDataExported.php:21]: Use typehints instead of docblocks

File: src/Notifications/PersonalDataExported.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+    /**
+     * @var \Illuminate\Support\Carbon 
+     */
+    public $deletionDatetime;
+
+    /**
+     * Create a notification instance.
```
Comment (freekmurze) [inline src/Notifications/PersonalDataExported.php:29]: Remove this and all other docblock, that don't add any meaniful info.

**Review by freekmurze — CHANGES_REQUESTED** [review]
It seems the tests are failing, could you take a look at that?

**Comment by ZeProf2Code** [comment]
It seems the tests are failing because of a bug in L7 prefer-lowest.

**Comment by freekmurze** [comment]
Feel free to update the lower version to make it work

**Pull request merged**